.github/workflows/e2e-tests.yml

```diff
@@ -222,10 +222,23 @@ jobs:
           echo "BRANCH=$BRANCH" >> "$GITHUB_ENV"
           echo "RUN_ID=$GITHUB_RUN_ID" >> "$GITHUB_ENV"
 
+      - name: Get PR title
+        uses: actions/github-script@v6
+        id: pr-title
+        with:
+          result-encoding: string
+          script: |
+            const { data: { title } } = await github.rest.pulls.get({
```

Comment:
Is this making an api call to get the PR title? Can we use the title in the github event to avoid an unnecessary call?

Reply:
Yes, that feels unnecessary, but I wasn't sure how to get the title in the GitHub event without running into the exact same issue we're trying to fix.